packages/axelar-soroban-std-derive/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[alias]
+wasm = "build --release --target wasm32-unknown-unknown"
```

Comment:
How does this alias work? if it's something like `cargo wasm`, could you update README as well?